### DISCUSSION TITLE
renovatebot.com and docs.renovatebot.com aren't available

### DISCUSSION THREAD

**Discussion by suzuki-shunsuke** [renovatebot/renovate#15279] [Request Help] [answered]
https://renovatebot.com
https://docs.renovatebot.com

<img width="977" alt="image" src="https://user-images.githubusercontent.com/13323303/164890236-374c74d2-6e54-4258-a4f5-28c408bd5e47.png">

```console
$ nslookup docs.renovatebot.com
Server:		192.168.11.1
Address:	192.168.11.1#53

** server can't find docs.renovatebot.com: NXDOMAIN
```

```console
$ nslookup renovatebot.com 
Server:		192.168.11.1
Address:	192.168.11.1#53

Non-authoritative answer:
*** Can't find renovatebot.com: No answer
```

https://www.nslookup.io/dns-records/renovatebot.com

<img width="1211" alt="image" src="https://user-images.githubusercontent.com/13323303/164891248-8d2e806f-42f8-43fd-9353-c1e70559d9e8.png">

<img width="1192" alt="image" src="https://user-images.githubusercontent.com/13323303/164891257-c623ea56-0e54-450f-8d74-a72e3e247dfc.png">

**Comment by rarkins** [comment, marked as answer]
We lost some DNS settings after a domain renewal problem. The app CNAME should have already been working, while docs is now also set and should propagate soon. The main page/domain is just a redirect

**Reply by suzuki-shunsuke** [reply]
Thank you for your reply.
https://docs.renovatebot.com/ has been recovered.
On the other hand, https://renovatebot.com/ is still unavailable.
Both A record and CNAME record aren't found. https://www.nslookup.io/dns-records/renovatebot.com

<img width="1239" alt="image" src="https://user-images.githubusercontent.com/13323303/164894559-149baae3-a0ce-4555-bcad-6209e59a8c66.png">

**Reply by suzuki-shunsuke** [reply]
I found https://renovatebot.com is redirected to https://www.whitesourcesoftware.com/free-developer-tools/renovate/ .
This issue has been solved.
